src/XliffTasks/build/XliffTasks.targets

```diff
@@ -40,6 +40,18 @@
     <_IsInnerXlfBuild Condition="'$(TargetFrameworks)' != '' and '$(TargetFramework)' != ''">true</_IsInnerXlfBuild>
   </PropertyGroup>
 
+  <!-- Suppress execution in certain contexts -->
```

Comment:
Why not simply add these checks to the condition https://github.com/dotnet/xliff-tasks/pull/234/files#diff-63bac777a3a8d84e48ca794beb80f52eR72, instead of conditionally set the targets?

Reply:
I'm indifferent. This approach is more clearly documented. If someone feels strongly though I could change it.

Reply:
It ends up being pretty ugly, especially as `BuildingProject` has to be conditioned with an `or` for the case that it's undefined:

```
          Condition="'$(EnableXlfLocalization)' == 'true' and '$(UpdateXlfOnBuild)' == 'true' and '$(_IsInnerXlfBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true' and '$(DesignTimeBuild)' != 'true' and ('$(BuildingProject)' == '' or '$(BuildingProject)' == 'true')"
```

I'm now thinking I'll make a property for the `Condition` rather than `BeforeTargets` attribute.

Reply:
Updated.